### ISSUE TITLE
scrollContainer doesn't work

### ISSUE THREAD

**Issue by liufeisprit** [twobin/react-lazyload#264]
when i need partial scroll ， and i set the scrollContainer  is partialScroll's dom node . like document.querySelector('....') . but is doesn't work . It only render first part .

![image](https://user-images.githubusercontent.com/25600325/67459095-54222180-f66a-11e9-9b97-97b3355ce7fd.png)
![image](https://user-images.githubusercontent.com/25600325/67459143-71ef8680-f66a-11e9-86a2-cc8c53a2d2db.png)

**Comment by jamesrx** [comment]
See #260

**Comment by liufeisprit** [comment]
It is I used the wraong anout scrollContainer? could i see the demo?

**Comment by liplylie** [comment]
you don't need to use `document.querySelector` in scrollContainer. Just pass the string of the className like this `scrollContainer={'.listView}

**Comment by liufeisprit** [comment]
> you don't need to use `document.querySelector` in scrollContainer. Just pass the string of the className like this `scrollContainer={'.listView'}`

yes , i used `scrollContainer={'.listView'}`  . But it only render first part , the next pages dosen't render real dom
